akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpMessage.scala

```diff
@@ -148,12 +148,16 @@ object HttpMessage {
       case HttpProtocols.`HTTP/1.0` ⇒ connectionHeader.isEmpty || !connectionHeader.get.hasKeepAlive
     }
 
-  class DiscardedEntity(f: Future[Done]) extends akka.http.javadsl.model.HttpMessage.DiscardedEntity {
+  /**
+   * Result of draining entity stream
+   */
+  final class DiscardedEntity(f: Future[Done]) extends akka.http.javadsl.model.HttpMessage.DiscardedEntity {
```

Comment:
Why not the comments I suggested? :)
https://github.com/akka/akka/pull/20707#discussion_r66443369